bc_obps/registration/api/user_operator.py

```diff
@@ -63,7 +66,7 @@ def get_user_operator(request, user_operator_id: int):
 
 
 @router.put(
-    "/select-operator/user-operator/{int:user_operator_id}",
+    "/select-operator/request-access/user-operator/{int:user_operator_id}",
```

Comment:
I think there still might be a rebasing issue here it looks like the route was changed